src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml

```diff
@@ -67,4 +69,42 @@
             </form>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+@section HeadJS
+{
+<script>
+    window.EsfaRecruit.PcaConfig = {
+        key: '@PcaConfig.Value.Key',
+        findEndpoint: '@PcaConfig.Value.FindEndpoint',
+        retrieveEndpoint: '@PcaConfig.Value.RetrieveEndpoint'
+    };
+</script>
+}
+@section FooterJS
+{
+<script asp-condition="@Model.HasMoreThanOneOrganisation">
+    var addresses = @Json.Serialize(Model.Organisations);

```

Comment:
Should we move `addresses` & `setAddresssById` out of global? Maybe move these into the `$(document).ready` function?

Reply:
👍